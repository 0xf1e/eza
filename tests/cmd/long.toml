bin.name = "eza"
args = "--long --no-user --no-time --no-filesize"
