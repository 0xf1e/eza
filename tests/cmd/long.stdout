.rw-r--r-- build.rs
.rw-r--r-- Cargo.lock
.rw-r--r-- Cargo.toml
.rw-r--r-- CHANGELOG.md
.rw-r--r-- cliff.toml
.rw-r--r-- CODE_OF_CONDUCT.md
drwxr-xr-x completions
.rw-r--r-- CONTRIBUTING.md
.rw-r--r-- deb.asc
.rw-r--r-- deny.toml
drwxr-xr-x devtools
drwxr-xr-x dump
.rw-r--r-- flake.lock
.rw-r--r-- flake.nix
.rw-r--r-- Justfile
.rw-r--r-- LICENCE
drwxr-xr-x man
.rw-r--r-- README.md
.rw-r--r-- rust-toolchain.toml
.rw-r--r-- screenshots.png
.rw-r--r-- SECURITY.md
drwxr-xr-x snap
drwxr-xr-x src
drwxr-xr-x target
drwxr-xr-x tests
.rw-r--r-- treefmt.nix
.rw-r--r-- Vagrantfile
drwxr-xr-x xtests
